.github/workflows/windows.yml

```diff
@@ -18,13 +18,19 @@ jobs:
         with:
           ruby-version: ${{ matrix.ruby }}
           architecture: 'x64'
+      - name: install dependcnies
+        run: choco install libreoffice-still
       - name: Update gems
         shell: pwsh
         run: |
           gem install bundler -v "~> 2"
           bundle config --local path vendor/bundle
           bundle update
           bundle install --jobs 4 --retry 3
+      - name: try to set path var
+        run: echo "::add-path::C:\Program Files\LibreOffice\program"
```

Comment:
hm, how that works? will this line really modify PATH?

Reply:
Yes, it is, its magic comment for github actions - https://help.github.com/en/actions/automating-your-workflow-with-github-actions/development-tools-for-github-actions#add-a-system-path-add-path